config/webpack/development.js

```diff
@@ -5,8 +5,8 @@ process.env.NODE_ENV = process.env.NODE_ENV || 'development'
 const environment = require('./environment')
 
 environment.config.devServer.https = {
-  key: fs.readFileSync('./config/localhost/https/localhost.key'),
-  cert: fs.readFileSync('./config/localhost/https/localhost.crt')
+  key: './config/localhost/https/localhost.key',
+  cert: './config/localhost/https/localhost.crt'
```

Comment:
Does this fix the slow tests? I thought the problem is that they don't exist (they have to be generated), not that they're in the wrong place.

Reply:
> Does this fix the slow tests? 30 secs
![image](https://user-images.githubusercontent.com/470137/64289791-c3c63b00-cf5c-11e9-95f5-1f8c70f01fa9.png)

> I thought the problem is that they don't exist (they have to be generated), not that they're in the wrong place.

@tvararu can tell you the nightmare

that dot in `.`/config/localhost/https/localhost.key is not what you think it is...